README.md

```diff
@@ -1,115 +1,500 @@
+Issue: When running /opt/rocm/bin/rocprof --hip-trace <output filename>, a warning message is printed to console: "Tool lib "/opt/rocm/roctracer/tool/libtracer_tool.so" failed to load", and no output file is generated, on systems with RHEL distro.
+
+Resolution: You can use either of the following workarounds to fix the issue:
+
+• Run Idconfig
+
+	'SUDO LDCONFIG' 
+
+ or 
+ 
+• Configure LD_LIBRARY_PATH  
+
+	'EXPORT LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:/OPT/ROCM/ROCTRACER/LIB'
+
+
+## Deprecated Features
+The following features are deprecated in the AMD ROCm v2.10 release. 
+
+### ROCm OpenCL Driver
+The AMD ROCm-OpenCL-Driver is now deprecated. Users should migrate to ROCm-CompilerSupport, which provides more comprehensive functionality. The compiler support repository provides various lightning compiler-related services. It currently contains a single library, the Code Object Manager (Comgr) at lib/comgr.
+
+ROCm-OpenCL-Driver will no longer be actively maintained after the v2.10 release. If your application was developed with the ROCm-OpenCL-Driver, we would encourage you to switch to the ROCm-CompilerSupport repository.
+
+### Peer-to-Peer Bridge Driver for PeerDirect
+The Peer-to-Peer bridge driver for the PeerDirect feature still works in the current release, however, it is now included as part of the ROCk kernel driver. ROCmRDMA allows third-party kernel drivers to utilize DMA access to the GPU memory. It allows a direct path for data exchange (peer-to-peer) using the standard features of PCI Express.
+
+Currently, ROCmRDMA provides the following benefits:
+
+•	Direct access to ROCm memory for 3rd party PCIe devices
+
+•	Support for PeerDirect(c) interface to offloads the CPU when dealing with ROCm memory for RDMA network stacks
+
+## Deploying ROCm
+AMD hosts both Debian and RPM repositories for the ROCm v2.10x packages. 
+
+The following directions show how to install ROCm on supported Debian-based systems such as Ubuntu 18.04. 
+
+Note: These directions may not work as written on unsupported Debian-based distributions. For example, newer versions of Ubuntu may not be compatible with the rock-dkms kernel driver. In this case, you can exclude the rocm-dkms and rock-dkms packages.
+
+For more information on the ROCm binary structure, see
+https://github.com/RadeonOpenCompute/ROCm/blob/master/README.md#rocm-binary-package-structure
+
+For information about upstream kernel drivers, see the Using Debian-based ROCm with Upstream Kernel Drivers section.
+
+## Ubuntu 
+### Installing a ROCm Package from a Debian Repository
+To install from a Debian Repository:
+1.	Run the following code to ensure that your system is up to date:
+      
+     	sudo apt update
+ 
+     	sudo apt dist-upgrade
+    
+     	sudo apt install libnuma-dev
+    
+     	sudo reboot 
+          
+    
+ 2. Add the ROCm apt repository.
+
+     For Debian-based systems like Ubuntu, configure the Debian ROCm repository as follows:
+   
+      	wget -q0 –
```

Comment:
Formatting looks is totally distorted here. The users who would copy-paste this onto their command line will be surprised.